src/Quadmath.jl

```diff
@@ -355,6 +355,11 @@ end
     Float128(@ccall(libquadmath.hypotq(x::Cfloat128, y::Cfloat128)::Cfloat128))
 @assume_effects :foldable atan(x::Float128, y::Float128) =
     Float128(@ccall(libquadmath.atan2q(x::Cfloat128, y::Cfloat128)::Cfloat128))
+
+Base.Integer(x::Float128) = Int64(x)
```

Comment:
To be consistent with Base?:
```suggestion
Base.Integer(x::Float128) = Int(x)
```